src/org/way2it/data_structures/list/List.java

```diff
@@ -1,4 +1,6 @@
-package org.way2it.data_structures.list;
+package List;
```

Comment:
Java naming convention says you should not use capital letters in package naming